src/databases/functions/users/user-set-karma-score.js

```diff
@@ -1,10 +1,34 @@
 const {User} = require('../../models');
+const {QueryTypes} = require('sequelize');
+const db = require('../../models/');
 
 module.exports = async (userId, combined_user_score, karma_score) => {
   if (!userId) return null;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function does not handle the case when `combined_user_score` or `karma_score` is not provided. It would be better to add checks for these parameters as well.
```diff
- if (!userId) return null;
+ if (!userId || combined_user_score === undefined || karma_score === undefined) return null;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->